contracts/src/v0.7/dev/KeeperRegistrarDev.sol

```diff
@@ -46,16 +46,15 @@ contract KeeperRegistrarDev is TypeAndVersionInterface, ConfirmedOwner {
   /**
    * @notice versions:
    * - KeeperRegistrar 1.1.0: Add functionality for sender allowlist in auto approve
+   *                        : Remove rate limit and add max allowed for auto approve
    * - KeeperRegistrar 1.0.0: initial release
    */
   string public constant override typeAndVersion = "KeeperRegistrar 1.1.0";
 
   struct AutoApprovedConfig {
     AutoApproveType configType;
-    uint16 allowedPerWindow;
-    uint32 windowSizeInBlocks;
-    uint64 windowStart;
-    uint16 approvedInCurrentWindow;
+    uint32 maxAllowed;
+    uint32 approvedCount;
```

Comment:
can we move `minLinkJules` into this struct? I think we could also change size to a `uint96` (same as on registry)

Reply:
It's semantically not part of 'autoApproveConfig'. How about I generalise it to 'config' and also move s_keeperRegistry there?

Reply:
Done